front/app/containers/Admin/projects/project/messaging/All/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+
+import {
+  Icon,
+  Box,
+  Title,
+  Text,
+  fontSizes,
+  colors,
+} from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+import useProjectCampaigns from 'api/campaigns/useProjectCampaigns';
+import { isDraft } from 'api/campaigns/util';
+
+import DraftCampaignRow from 'components/admin/Email/DraftCampaignRow';
+import SentCampaignRow from 'components/admin/Email/SentCampaignRow';
+import { ButtonWrapper } from 'components/admin/PageWrapper';
+import Pagination from 'components/admin/Pagination';
+import { List } from 'components/admin/ResourceList';
+
+import { FormattedMessage } from 'utils/cl-intl';
+import { getPageNumberFromUrl } from 'utils/paginationUtils';
+
+import messages from '../messages';
+
+import NewCampaignButton from './NewCampaignButton';
+
+const NoCampaignsWrapper = styled.div`
```

Comment:
Would be nice to rewrite these all to `Box`, `Title` and `Text` components (also in the `CustomEmails/All/index` file)